bigquery/snippets/job/bigquery_copy_table_multiple_source.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package job
+
+// [START bigquery_copy_table_multiple_source]
+
+import (
+	"cloud.google.com/go/bigquery"
+	"context"
+	"fmt"
+)
+
+func copyMultiTable(projectID, srcDatasetID string, srcTableIDs []string, dstDatasetID, dstTableID string) error {
+	// projectID := "my-project-id"
+	// srcDatasetID := "sourcedataset"
+	// srcTableIDs := []string{"table1","table2"}
+	// dstDatasetID = "destinationdataset"
+	// dstTableID = "destinationtable"
+	ctx := context.Background()
+	client, err := bigquery.NewClient(ctx, projectID)
+	if err != nil {
+		return fmt.Errorf("bigquery.NewClient: %v", err)
+	}
+
+	srcDataset := client.Dataset(srcDatasetID)
+	dstDataset := client.Dataset(dstDatasetID)
+	var tableRefs []*bigquery.Table
```

Comment:
Perhaps just `tables`?

Reply:
This is just the reference id for the table, not the metadata/data.  Wanted to remain clear these are identifiers only.